backup-recovery.adoc

```diff
@@ -1,26 +1,101 @@
-Make sure that you merge and replace all the existing files. Importing the backup means that you delete the current state of the wallet, so make sure you have a second backup of the status quo. Upon restart of the Bisq client, the the backup account will automatically load.
+* macOS: `/Users/<username>/Library/Application Support/Bisq`
+* Linux: `/home/<username>/.local/share/Bisq`
+* Windows 7+: `C:\Documents and Settings\<username>\AppData\Roaming\Bisq`
+* Windows XP: `C:\Documents and Settings\<username>\Application Data\Bisq`
+
+**The safest, quickest, easiest, and most comprehensive thing you can do is to back up this whole folder at regular intervals.** You can either close Bisq and copy this folder to your backup location, or you can export a backup directly to your backup location while Bisq is running from the `Backup` tab of the `Account` screen.
+
+NOTE: Backup files are not encrypted. If you've set a wallet password, your wallet files will be encrypted, but all your other data will not be. We recommend encrypting the whole backup folder with something like gpg, 7-Zip, Cryptomator, etc.
+
+=== Restore
+
+First, make sure the Bisq program is closed. And just in case something goes wrong with the restore, back up the current state of your data directory (i.e., the `Bisq` folder).
+
+Then delete the whole `Bisq` folder and replace it with the `Bisq` folder from your backup.
+
+=== Details
+
+In your Bisq data directory, you will see a few items. The most notable file is `bisq.log`, which is the Bisq program log file. Checking it can be helpful when you have issues with the Bisq program.
+
+The other notable item is `btc_mainnet`, which is the main folder containing data for your payment accounts, Tor setup, and wallet.
+
+== Payment Accounts
+
+There are 2 components to payment accounts on Bisq: account details and account aging.
+
+**Account details** consist of all the payment methods you've set up and their associated account numbers, bank information, addresses, etc. You can export this data from within Bisq on the `National currency accounts` and `Altcoin accounts` tabs of the `Account` screen. Click `Export accounts` and Bisq will output a file that you can use to import these details later.
+
+**Account aging** establishes the age of your payment accounts. To reduce fraud, newer payment accounts have lower trading limits.
+
+To include account aging with your account details import, replace the `btc_mainnet/keys/sig.key` file with the `sig.key` file from your backup. Be sure you don't have any open offers, trades, or disputes before changing `sig.key`, otherwise you won't be able to respond to those activities.
+
+If you don't have an accounts export file, but you have a full backup of the data directory, you can still fully restore your accounts details and accounts aging by manually copying these files to your data directory from a backup:
+
+* `btc_mainnet/db/AccountAgeWitnessStore` (salt)
```

Comment:
AccountAgeWitnessStore is not needed. That contains the public hashes but not the salt which is part of the payment account data.